packages/nextjs/app/page.tsx

```diff
@@ -1,16 +1,36 @@
 "use client";
 
+import { useEffect, useState } from "react";
 import Link from "next/link";
 import type { NextPage } from "next";
+import { usePublicClient } from "wagmi";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
-import { useScaffoldContractRead } from "~~/hooks/scaffold-eth";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldContractRead, useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
 
 const Home: NextPage = () => {
+  const [checkedInBuilders, setCheckedInBuilders] = useState<string[]>([]);
+  const [isConnectedOptimism, setIsConnectedOptimism] = useState<boolean>(false);
+
+  //HOOKS
+
+  const { chain } = usePublicClient();
```

Comment:
Lets remove this checkout first section of https://github.com/BuidlGuidl/batch3.buidlguidl.com/pull/14#issuecomment-1988806853